### DISCUSSION TITLE
Grouping releases

### DISCUSSION THREAD

**Discussion by Urigo** [graphcommerce-org/graphcommerce#1673] [Ideas]
We recently improved our release process in @the-guild-org and on GraphQL Mesh with new contributions @dotansimha made for the changesets project.

It makes it possible to group releases, make them easier to track and makes less GitHub notifications.
I was thinking maybe it's something you would also want to use:

https://github.com/Urigo/graphql-mesh/releases

Here are the details (and you night find there more useful tools you might like):
https://github.com/the-guild-org/#meta---tools-we-use-to-build-our-tools

**Comment by paales** [comment]
Saw this post as soon as you posted. This has been on the top of my mind for a long time, thanks for the details @Urigo how you guys are implementing it. Taking a detailed look now.
